queries/html_tags/injections.scm

```diff
@@ -27,23 +27,23 @@
       (#not-match? @_no_type_lang "\\s(lang|type)\\s*\\=")
     (raw_text) @javascript))
 
-(
-  (script_element
-    (start_tag
-      (attribute
-        (attribute_name) @_type
-        (quoted_attribute_value (attribute_value) @_javascript)))
-    (raw_text) @javascript)
-  (#eq? @_type "type")
-  (#any-of? @_javascript "text/javascript" "module")
-)
+; <script type="mimetype-or-well-known-script-type">
+(script_element
+   (start_tag
+      ((attribute
+           (attribute_name) @_attr (#eq? @_attr "type")
+           (quoted_attribute_value (attribute_value) @_type))))
+   (raw_text) @language (#set-lang-from-mimetype! @_type))
```

Comment:
Shouldn't this be content instead of language?

Reply:
This kinda makes me wonder, how was the test still passing before, if `@content` was never assigned to a node?

Reply:
i'm guessing it's becaue `metadata.language` is set in the directive

Reply:
Is this resolved? I'll merge if so

Reply:
It's resolved as of latest commit